lib/minimap.js

```diff
@@ -1108,60 +1118,53 @@ export default class Minimap {
 
   /**
    *  get the DecorationManagement API for the current minimapElement
-   * @return {DecorationManagement}
+   * @return {DecorationManagement | null | undefined}
    */
   getDecorationManagement() {
     if (!this.DecorationManagement) {
-      if (this.minimapElement) {
+      if (this.minimapElement?.DecorationManagement) {
         this.DecorationManagement = this.minimapElement.DecorationManagement
-      } else {
-        // TODO: find why this becomes null: https://github.com/atom-minimap/minimap/issues/766
-        throw new Error(`getDecorationManagement failed.
-       this.DecorationManagement: ${JSON.stringify(this.DecorationManagement)}
-       this.minimapElement: ${JSON.stringify(this.minimapElement)}
-       this.textEditor: ${this.getTextEditor()}
-       this: ${JSON.stringify(this)}
-     `)
       }
+      return null
```

Comment:
why are we returning `null` here after we set `this.DecorationManagement`?

Reply:
Oops. I forgot to put this inside `else`. It is not needed anyway. I removed it.